inc/html.class.php

```diff
@@ -5551,6 +5551,7 @@ static function redefineAlert() {
                $(this).parent().prev('.ui-widget-overlay').addClass('glpi_modal');
             },
             close: function(){
+               $(this).parent().prev('.ui-widget-overlay').removeClass('glpi_modal');
```

Comment:
hum... this looks like something added from jquery-ui, I do not understand why you have to remove it explicitely

Reply:
Indeed, I don't remember changing this, it must be a mistake.
Fixed in latest commit.